.github/workflows/rust.yml

```diff
@@ -9,14 +9,17 @@ jobs:
     steps:
     - name: Checkout
       uses: actions/checkout@v1
+    - name: Install runt
+      uses: actions-rs/cargo@v1
+      with:
+        command: install
+        args: runt
```

Comment:
Maybe explore rust's `dev-dependencies` section in the `Cargo.toml` instead of installing it here. idk enough about the dev dependendies thing to know if it will be enough to get this to work.

Reply:
From what I can tell dev-dependencies are used for compiling tests. Not sure how I can force it to install runt as a script tool.